### PR TITLE
fix: Handle import of common document name (JSON/XML) by using data source ID a…

### PR DESCRIPTION
**Pull request by pleacu**
…nd doc ID - also add doc desc.

Fixes: #2198
ENTESB-12361

### CODE REVIEW

File: ui/packages/atlasmap-core/src/services/initialization.service.ts [hunk 1]
```diff
@@ -650,7 +657,11 @@ ${error.status} ${error.statusText}`,
     inspectionParameters?: { [key: string]: string }
   ): DocumentDefinition {
     const model: DocumentInitializationModel = new DocumentInitializationModel();
-    model.id = name;
+    model.name = name;
+    model.id =
```
Comment (igarashitm) [inline ui/packages/atlasmap-core/src/services/initialization.service.ts:661]: I'd assing a GUID to allow mutliple same-type&same-name documents.

Comment (pleacu) [reply ui/packages/atlasmap-core/src/services/initialization.service.ts:661]: Are we sure we want to allow multiple documents with the exact same name to be imported into the panels?  The user wanted `address.json` and `address.xml` to be allowed.  I don't see a demand for 100 `address.json` documents and adding a global unique ID will add to the complexity.

Comment (igarashitm) [reply ui/packages/atlasmap-core/src/services/initialization.service.ts:661]: Yes that has to be possible. For example retrieving address object twice but with a different condition, where schema is same.

**Comment by pleacu** [comment]
Making a few more changes than just an update to the old PR - will resubmit

**Pull request closed without merging**